src/client/initialize.ts

```diff
@@ -126,28 +125,48 @@ export function initialize<T = {}>(): PluginInstance<T> {
         on('config', listener);
         return () => off('config', listener);
       },
-      getVariable(id: string): WorkbookVariable {
-        return subscribedWorkbookVars[id];
+      getVariable(configId: string) {
+        if (configId === undefined) {
+          console.warn(`Invalid config variable: ${configId}`);
```

Comment:
I wonder if we should mark these as deprecation warnings and indicate that the ability to do this will be removed in the next major version?
```suggestion
          console.warn(`Deprecation warning: variable ID must be defined. The ability to pass an undefined ID will be removed in the next major version`);
```

Reply:
I think an error like this might lead new users to believe that having a defined config id _is_ currently valid, but part of the issue with throwing here in a future version, is that when their app is not embedded into slate, it will throw an error from the config not being defined. This could be acceptable behavior, but I personally like the softer warning when the config isn't defined.

That being said, I think a good expansion to this in the future is to potentially rework the config system so that we can validate that a config value corresponds to a config option of the right type instead of just ensuring it's not undefined.